chatbot.html

```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
 //# sourceURL=pen.js
     </script>
 </main>
+
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300&display=swap" rel="stylesheet">
 <style>
     article{
-        background: url('body_wallpaper.png') no-repeat fixed center;
+        background: url('website_images/body_wallpaper.png') no-repeat fixed center;
+        font-family: "Roboto Slab", sans-serif;
+        font-style: normal;
+        font-size: 20px;
         
     }
     .content-background{
         background-color: silver;
-        opacity: 0.8;
+        opacity: 0.9;
         padding:20px
     }
+    
```

Comment:
Added styling for the tabs and slides, using old brown paper design to match the theme. 
For tab content, there is a styled description using the old brown paper theme as well.